ui/src/main/webapp/src/app/components/navigation/breadcrumbs.component.html

```diff
@@ -1,6 +1,42 @@
+
+<style>
+ol.breadcrumb {
+    /* Pull the line to the borders. */
+    margin-left:  -30px;
+    margin-right:  -30px;
+    padding-left: 30px;
+    padding-right: 30px;
+    border-bottom: 1px solid #d5d5d5;
+    margin-bottom: 0px;
+    background-color: white;
+}
+
+ol.breadcrumb > li {
+    font-size: 10pt;
+}
+
+ol.breadcrumb > li:last-child > a,
+ol.breadcrumb > li:last-child > a:hover {
+    font-weight: bold;
+    text-decoration: none;
+    color: dimgrey;
+    cursor: default !important;
+}
+</style>
+
+<style *ngIf="breadCrumbLinks?.length > 1">
```

Comment:
I think it works to do it this way (I've had to do it this way as a hack in wordpress before).

I can't think of a reason to do it like this in angular, though. I think it would be better in a separate file.